### ISSUE TITLE
Make pass a visible action in playback

### ISSUE THREAD

**Issue by andymarden** [online-go/online-go.com#2467]
When a pass occurs in the looking at a game (particularly in the correct of community moderation, nothing happens or is shown.

It would make it easier to understand if something appeared indicating a pass has occurred.

**Comment by BHydden** [comment]
Yeah maybe '$player-colour passes' could flash on the screen,  either over the board or around where "undo requested"  shows up.

**Comment by GreenAsJade** [comment]
At first I wondered why you can't just tell this from the game tree, which has a dot in it, but I think that the problem is that you're in a view where the game tree is not on the page eh (eg mobile).
